hudi-common/src/test/java/org/apache/hudi/common/util/TestBaseFileUtils.java

```diff
@@ -42,7 +43,7 @@ public void testGetColumnRangeInPartition() {
         "path/to/file2", COLUMN_NAME, 3, 8, 1, 15, 120, 250);
     List<HoodieColumnRangeMetadata<Comparable>> fileColumnRanges = Arrays.asList(fileColumnRange1, fileColumnRange2);
     // Step 2: Call the Method
-    HoodieColumnRangeMetadata<Comparable> result = FileFormatUtils.getColumnRangeInPartition(PARTITION_PATH, fileColumnRanges);
+    HoodieColumnRangeMetadata<Comparable> result = FileFormatUtils.getColumnRangeInPartition(PARTITION_PATH, fileColumnRanges, Collections.emptyMap());
```

Comment:
Let's UT the method for non-empty map and non-null min/max values where type promotion happens and type-coercion path is exercised.

Reply:
yes, I am currently writing them